### DISCUSSION TITLE
Asking about Contribution

### DISCUSSION THREAD

**Discussion by abhishekgujarathi** [spiral-project/ihatemoney#1095] [General]
How can i Contribute to this repo for translation..?

**Comment by Glandos** [comment]
From the README.md, there is a link to the contributions page: https://ihatemoney.readthedocs.io/en/latest/contributing.html
Here, you'll find a section regarding translations: https://ihatemoney.readthedocs.io/en/latest/contributing.html#as-a-translator
And there is a link to redirect you to the translation website: https://hosted.weblate.org/projects/i-hate-money/i-hate-money/

You should log in on Weblate (the translation platform we are using), and it's possible to do it with your GitHub account. After that, just click on the pencil in front of your language, or add a new language with the button at the bottom of the page.
